superset/contrib/docker/docker-entrypoint.sh

```diff
@@ -37,7 +37,7 @@ fi
 if [ "$#" -ne 0 ]; then
     exec "$@"
 elif [ "$SUPERSET_ENV" = "development" ]; then
-    celery worker --app=superset.sql_lab:celery_app --pool=gevent -Ofair &
+    celery worker --app=superset.sql_lab:celery_app --logfile=/tmp/celery.log --loglevel=INFO --pool=gevent -Ofair &
```

Comment:
It may worth to expose logfile and loglevel as env vars, WDYT?

Reply:
let's do it in separate PR